src/resources/measurements.ts

```diff
@@ -1,6 +1,57 @@
 import { Measurement, MeasurementsApi } from '../codegen/api';
 import { Configuration as APIConfiguration } from '../codegen/configuration';
-import { fixMeasurementFields } from '../utils';
+
+interface MeasurementInput extends Omit<Measurement, 'time'> {
+  time?: Date | string;
+}
+
+/**
+ * This helper augments the body with "meter_name" based on the meterName field.
+ *
+ * TODO: API expects "meter_name", while the generated type uses meterName.
+ * This function should be deleted once the API
+ *
+ * @param param0
```

Comment:
this is incorrect, i think? we haven't really been using docstrings, though it might be a good idea for OSS repos like our SDK repos.

Reply:
Ugh these `@param` and `@returns` got autogenerated when I made the header comment. I'll remove them.